redbot/cogs/permissions/permissions.py

```diff
@@ -819,7 +819,7 @@ def _load_rules_for(
                     cog_or_command.deny_to(model_id, guild_id=guild_id)
 
     def cog_unload(self) -> None:
-        self.bot.loop.create_task(self._unload_all_rules())
+        asyncio.create_task(self._unload_all_rules())
```

Comment:
Do we want to just bulk replace with `asyncio.create_task` or utilize the new async `cog_unload` where applicable instead?

Reply:
This PR is independent of d.py 2.0 changes so no changes related to it should be done here.